backend/fbdev.c

```diff
@@ -20,6 +20,7 @@
 
 #define FBDEV_NAME "FRAMEBUFFER"
 #define FBDEV_DEFAULT "/dev/fb0"
+#define FBIO_CACHE_SYNC 0x4630
```

Comment:
Is there any reference for `FBIO_CACHE_SYNC` and if it is defined by kernel headers?